src/ui/locales/de.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+const locale = {
+    "AttributionControl.ToggleAttribution": "Zuordnung umschalten",
+    "AttributionControl.MapFeedback": "Karten-Feedback",
+    "FullscreenControl.Enter": "Vollbildmodus aktivieren",
+    "FullscreenControl.Exit": "Vollbild beenden",
```

Comment:
I think this should be consistent with the one above. So, `Vollbildmodus beenden` or even more consistent `Vollbildmodus deaktivieren`. Alternatively, remove `modus` in both lines.